app/presenters/grove_monitor_presenter.rb

```diff
@@ -2,10 +2,16 @@ class GroveMonitorPresenter
   attr_reader :lost_students, :absent_students, :locations
   def initialize(grove)
     @absent_students = grove.students.absent
-    @lost_students = grove.students.lost
+    @lost_students = grove.students.lost.where(at_school: true)
+
```

Comment:
I don't think this where needed to be added. It should be on the Student model and not in the presenter.